src/dfx/src/commands/wallet/send.rs

```diff
@@ -24,13 +24,13 @@ pub async fn exec(env: &dyn Environment, opts: SendOpts) -> DfxResult {
     #[derive(CandidType)]
     struct In {
         canister: Principal,
-        amount: u64,
+        amount: u128,
     }
     let canister = Principal::from_text(opts.destination.clone())?;
     // amount has been validated by cycle_amount_validator
-    let amount = opts.amount.parse::<u64>().unwrap();
+    let amount = opts.amount.parse::<u128>().unwrap();
     let (res,): (Result<(), String>,) =
-        wallet_update(env, "wallet_send", In { canister, amount }).await?;
+        wallet_update(env, "wallet_send128", In { canister, amount }).await?;
```

Comment:
Should this go through the agent's wallet_send so that it can route to either `wallet_send` or `wallet_send128` ?